pkg/dependencies/factory.go

```diff
@@ -1317,10 +1317,18 @@ func (f *Factory) WithPackageInstaller(spec *cluster.Spec, packagesLocation, kub
 	return f
 }
 
-func (f *Factory) WithPackageControllerClient(spec *cluster.Spec, kubeConfig string) *Factory {
+// WithPackageInstallerWithoutWait builds a package installer that doesn't wait for active bundles.
+func (f *Factory) WithPackageInstallerWithoutWait(spec *cluster.Spec, packagesLocation, kubeConfig string) *Factory {
+	f.WithPackageInstaller(spec, packagesLocation, kubeConfig)
+	f.WithPackageControllerClient(spec, kubeConfig, curatedpackages.WithSkipWait())
```

Comment:
Shouldn't this be in the opposite order? You want your special package controller client to be built before the default one

```suggestion
	f.WithPackageControllerClient(spec, kubeConfig, curatedpackages.WithSkipWait()).
	    WithPackageInstaller(spec, packagesLocation, kubeConfig)
	
```